seed_data.sh

```diff
@@ -14,3 +14,4 @@ python manage.py loaddata payment
 python manage.py loaddata order
 python manage.py loaddata order_product
 python manage.py loaddata favoritesellers
+python manage.py loaddata recommendations
```

Comment:
Great! You can also add a line to seed your productrating data here as well! Not necessary for this PR, but maybe in a future one!

Reply:
I think there is a product rating file in there.  Maybe it got shuffled around in the git-verse. I'm make sure it is included in the updated branch.